cli/cmd/update.go

```diff
@@ -134,9 +134,13 @@ func updateStackMeta(ctx context.Context, stackName string, happyClient *HappyCl
 		"happy/meta/configsecret": happyClient.HappyConfig.GetSecretId(),
 	})
 	if sliceDefaultTag != "" {
-		happyClient.Tag = sliceDefaultTag
+		happyClient.ArtifactBuilder.WithTags([]string{sliceDefaultTag})
```

Comment:
Will this override the other tags?

Reply:
ArtifactBuilder is also used in the `happy push` command and that command allows you to specify multiple tags to push. So we need a way to specify multiple tags. In the update/create case though, you can only specify one tag, so that is why I check it here.

Reply:
I am going to work with @dtsai-czi on cleaning up the slices work. I think they haven't been working for a bit, so I will investigate this again then.